src/store.js

```diff
@@ -68,7 +72,8 @@ class Store {
    * Binds an action type => handler
    */
   on(actionType, handler) {
-    this.__handlers = this.__handlers.set(actionType, handler)
+    console.assert(actionType, "Store.on called without actionType");
```

Comment:
FWIW, the `console` API is supported by all the major browser engines.  Maybe it's not formally specified?

I'm used to using assertions to verify the state of the world, so I didn't even really consider whether an assertion or a conditional error was more appropriate when I wrote that.  I don't have a strong opinion either way, so it's now a conditional error.

Just now, I quickly tested the differences between the two.  It looks like errors thrown by `console.assert` break out of `try/catch`.  So, that's a good difference to know about…